### PR TITLE
boards: remove deleted modules

### PR DESCRIPTION
**Pull request by dagar**

### CODE REVIEW

File: boards/px4/fmu-v6x/default.px4board [hunk 1]
```diff
@@ -6,7 +6,6 @@ CONFIG_BOARD_SERIAL_GPS2="/dev/ttyS7"
 CONFIG_BOARD_SERIAL_TEL1="/dev/ttyS6"
 CONFIG_BOARD_SERIAL_TEL2="/dev/ttyS4"
 CONFIG_BOARD_SERIAL_TEL3="/dev/ttyS1"
-CONFIG_BOARD_SERIAL_EXT2="/dev/ttyS3"
```
Comment (dagar) [inline boards/px4/fmu-v6x/default.px4board:9]: @baumanta CONFIG_BOARD_SERIAL_EXT2 keeps getting stripped out of v6x, I'm not sure what's missing.

**Comment by PetervdPerk-NXP** [comment]
Not sure how you're updating this.
But there's a make command to do this automatically and also solves the ordering
`make updateconfig`
Might want to run a pass from that over this PR.

**Comment by dagar** [comment]
Thanks @PetervdPerk-NXP

**Comment by dagar** [comment]
Replaced with automatic PX4Buildbot update. https://github.com/PX4/PX4-Autopilot/pull/20869

**Pull request closed without merging**